apps/ui/src/queries/votingPower.ts

```diff
@@ -0,0 +1,153 @@
+import { utils } from '@snapshot-labs/sx';
+import { useQuery } from '@tanstack/vue-query';
+import { MaybeRefOrGetter } from 'vue';
+import { getNetwork, offchainNetworks, supportsNullCurrent } from '@/networks';
+import { VotingPower } from '@/networks/types';
+import { NetworkID, Proposal, Space } from '@/types';
+
+export type VotingPowerItem = {
+  votingPowers: VotingPower[];
+  symbol: string;
+  canVote: boolean;
+};
+
+type Snapshot = Proposal['snapshot'] | null;
+
+type SpaceLike = Pick<
+  Space,
+  'id' | 'network' | 'snapshot_chain_id' | 'voting_power_symbol'
+>;
+
+const CACHE_TTL = 1000 * 60;
+
+const VOTING_POWER_KEYS = {
+  space: (
+    block: MaybeRefOrGetter<Snapshot>,
+    spaceId: MaybeRefOrGetter<string>
+  ) =>
+    ['votingPower', toRef(() => web3.value.account), block, spaceId] as const,
```

Comment:
I'm thinking that we should make account a argument to this key function instead of making it implicit like this. Might be useful when we invalidate queries.